frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/capacity-and-nodes.tsx

```diff
@@ -68,6 +70,33 @@ type SelectNodesDetailsProps = {
   memory: number;
 };
 
+export const EnableTaintNodes: React.FC<enableTaintNodesProps> = ({ state, dispatch, mode }) => {
```

Comment:
```suggestion
export const EnableTaintNodes: React.FC<EnableTaintNodesProps> = ({ state, dispatch, mode }) => {
```

Reply:
@afreen23 The other parts of the file doesn't have this kind of case convention, I think.

Reply:
I see one component, rest are all good.
```
export const SelectNodesText: React.FC<SelectNodesTextProps> 
export const SelectNodesDetails: React.FC<SelectNodesDetailsProps>
export const StretchClusterFormGroup: React.FC<stretchClusterFormGroupProps> -> fix this as well
```